pkg/api/azenvtypes.go

```diff
@@ -134,14 +134,23 @@ var (
 		ImageVersion:   "2020.12.02",
 	}
 
-	// AKSWindowsServer2019OSImageConfig is the AKS image based on Windows Server 2019
+	// AKSWindowsServer2019OSImageConfig is the aks-engine image based on Windows Server 2019
 	AKSWindowsServer2019OSImageConfig = AzureOSImageConfig{
 		ImageOffer:     "aks-windows",
 		ImageSku:       "2019-datacenter-core-smalldisk-2012",
 		ImagePublisher: "microsoft-aks",
 		ImageVersion:   "17763.1637.201210",
 	}
 
+	// AKSWindowsServer2019ContainerDOSImageConfig is the aks-engine image based on Windows Server 2019
+	// configured with containerd
+	AKSWindowsServer2019ContainerDOSImageConfig = AzureOSImageConfig{
+		ImageOffer:     "aks-windows",
+		ImageSku:       "2019-datacenter-core-ctrd-2012",
```

Comment:
Does it mean that aks-engine will use a different VHD for containerd support? May I ask the reason not using the same 2019 VHD?

Reply:
I guess this is for differentiating docker windows VHD, otherwise, this can only be told from the image version which is not obvious

Reply:
The main reason we want to use two different VHDs is because docker and containerd store container images differently (both in a different format and at a different location on disk). 
If we wanted to use the same VHD we would need to prepull nanoserver/servercore/other images with both docker and containerd and this would not leave much room left on the VHD.

Reply:
@marinred I think that we should do it, too.